lib/utils/validateOptions.js

```diff
@@ -42,7 +42,12 @@ module.exports = function (result, ruleName, ...optionDescriptions) {
 		result.warn(message, {
 			stylelintType: 'invalidOption',
 		});
-		_.set(result, 'stylelint.stylelintError', true);
+		result.stylelint = result.stylelint || {
+			disabledRanges: {},
+			ruleSeverities: {},
+			customMessages: {},
+		};
```

Comment:
[note] I'm a bit concerned about the newly added properties like `disabledRanges`, but it seems due to our TypeScript definitions (maybe another issue). 🤔 

Anyway, I think no problems going on as-is. 👍🏼

Reply:
Yes, that was because of the TypeScript definitions. It wouldn't let me just set `result.stylelint` to `{}`.